src/utils/Configuration.ts

```diff
@@ -133,6 +133,14 @@ class Configuration {
         return this.config.systemNumberInitialValue;
     }
 
+    /**
+     * Retrieves the plate serial number initial value
+     * @returns any
+     */
+    public getPlateSerialNumberInitialValue(): any {
```

Comment:
suggest to make private

Reply:
cannot be private as it is called inside the NumberService class